jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -81,6 +81,22 @@
 	 */
 	public static boolean isWindows = false;
 
+	/**
+	 * True if operating system is z/OS.
+	 */
+	public static boolean isZOS = false;
```

Comment:
Can `isZOS` be `final` or accessed via `getter` instead? Similarly question for `isWindows`.

Reply:
Please arrange for the fields to be `final`.

Reply:
> Can `isZOS` be `final`? 

Done